.golangci.yml

```diff
@@ -10,26 +11,23 @@ linters-settings:
     values:
       regexp:
         company: .*
-        copyright-holder: Copyright \(c\) ({{year-range}}) {{company}}\n\n
+        copyright-holder: Copyright \(c\) ({{mod-year-range}}) {{company}}\n\n
```

Comment:
The only repository where this fix does not help to avoid goheader linter error for file, that has not been modified in the PR.

Reply:
@denis-tingaikin any ideas to fix golangci-lint error?
```
Error: Pattern (Copyright \(c\) (((20\d\d\-2023)|(2023))) .*\n\n)+ doesn't match. (goheader)
```
I can get it locally too in connection.go file

Reply:
ok, just update it